components/atom/button/src/index.scss

```diff
@@ -86,21 +86,30 @@ $p-atom-button-large: $p-l !default;
   }
 
   &-group {
+    margin-left: -1px;
+    margin-right: 0px;
     
     & + & {
       margin-left: -1px;
```

Comment:
Shouldn't it be `margin-left: -$bdrs-m;` ?
You should try to put $bdrs-m: 3px in your dev environment and have a look on how it behaves.

Reply:
Not really, this value achieves the needed effect of grouping the buttons while respecting original dimensions on each button